apps/task-master/.env.example

```diff
@@ -16,14 +16,14 @@ INFERENCE_SERVER="http://localhost:28664/"
 VECTOR_DB_SERVER="http://localhost:8000"
 
 # Document indexing and retrieval configs
-DOCS_INDEXING_CHUNK_SIZE=1000
+DOCS_INDEXING_CHUNK_SIZE=2800
```

Comment:
We shouldn't change the default value here as it will give error for people trying to use `text-embedding-ada-002`. Can you undo changes to `DOCS_INDEXING_CHUNK_SIZE` in all env file?